### DISCUSSION TITLE
Other PLC Communication`

### DISCUSSION THREAD

**Discussion by RagavL** [dmroeder/pylogix#153] [Q&A] [answered]
I understand that this library works only for Rockwell Automation based PLC.I am trying to connect to a B&R PLC.I could not find any other library.Can I still use this library?

**Comment by TheFern2** [comment, marked as answer]
Hi @RagavL. Nope, this is only for Rockwell PLCs, you'll need to find a library specific for your PLC brand as specified in the readme. Good luck!

These links below should give you an idea on how to read tags from B&R PLC and sorry for not having any more answers I've only dealt with Siemens and Allen Bradley PLCs.

https://github.com/search?q=b%26r+PLC
https://github.com/hilch/brwatch
